authorization-server/jwt-authorization-grant-token-exchange.js

```diff
@@ -4,8 +4,18 @@ import validatePresence from 'oidc-provider/lib/helpers/validate_presence.js';
 import instance from 'oidc-provider/lib/helpers/weak_cache.js';
 import { OAuthBadRequest, requestIdJwtAuthzGrant } from 'id-assert-authz-grant-client';
 import { getIdToken } from './utils/id-token-cache.js';
-
+import createRedisClient from './utils/redis.js';
 // eslint-disable-next-line import/prefer-default-export
+const saveTokenToRedis = async (token) => {
+  try {
+    const redisClient = await createRedisClient();
+    await redisClient.connect();
+    await redisClient.set('jag_subject_token', token);
```

Comment:
Wouldnt two requests override each other?  Seems like we need to key this off of a user id or something